homeassistant/components/august/__init__.py

```diff
@@ -93,6 +93,27 @@ async def async_setup_august(
     return True
 
 
+@callback
+def _async_trigger_ble_lock_discovery(
+    hass: HomeAssistant, locks_with_offline_keys: list[LockDetail]
+):
+    """Update keys for the yalexs-ble integration if available."""
+    for lock_detail in locks_with_offline_keys:
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                "yalexs_ble",
+                context={"source": SOURCE_INTEGRATION_DISCOVERY},
```

Comment:
Since this is an internal discovery, should this be done via a callbackc method exposed by `yalexs_ble` ?